stable/locust/templates/locust-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: locust-master
```

Comment:
* Name should us fullname template.
* Add standard labels.
* You may want to base this on the standard ingress template you get with `helm create` and Helm 2.8+